TypeCobol/Compiler/Directives/CompilerDirective.cs

```diff
@@ -867,15 +865,15 @@ public TextNameVariation(string textNameWithSuffix)
             /// <summary>
             /// Text name without suffix
             /// </summary>
-            public string TextName { get { return HasSuffix ? TextNameWithSuffix.Substring(0, 7) : TextNameWithSuffix; } }
+            public string TextName { get { return HasSuffix ? TextNameWithSuffix.Substring(0, 7).ToUpper() : TextNameWithSuffix; } }
```

Comment:
This change, means that the copy will now be written in uppercase in error message.
I'd rather set only properties `Suffix`and `PreSuffix` in uppercase. What do you think ?